auv_setup/launch/gladlaks.launch

```diff
@@ -15,9 +15,7 @@
   <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
   <include file="$(find motion_launch)/launch/motion.launch"            pass_all_args="true"/>
   <include file="$(find navigation_launch)/launch/navigation.launch"    pass_all_args="true"/>
-
-  <!-- Temporary -->
-  <node pkg="battery_monitor" type="battery_monitor.py" name="battery_monitor" output="screen"/>
+  <include file="$(find internal_status)/launch/internal_status.launch"/>
```

Comment:
This node will only work when running on the Xavier, so it should be included in the 'real' group